servant-exceptions/servant-exceptions.cabal

```diff
@@ -3,14 +3,14 @@ version:               0.1.1
 description:           Extensible exceptions for servant APIs
 homepage:              https://github.com/ch1bo/servant-exceptions#readme
 license:               BSD3
-license-file:          ../LICENSE
+license-file:          LICENSE
 author:                Sebastian Nagel
 maintainer:            sebastian.nagel@ncoding.at
 copyright:             2017 Sebastian Nagel
 category:              Web
 build-type:            Simple
-extra-source-files:    ../CHANGELOG.md
-                     , ../README.md
+extra-source-files:    CHANGELOG.md
+                     , README.md
```

Comment:
Just keep the single README.md in repository root and drop it from the package itself.

Reply:
You won't have any top-level README on hackage then...